### PR TITLE
entity-management: bump

### PR DESCRIPTION
**Pull request by genric**
Entity management version bump and its new dependencies.
Fix paths to ipyparallel executables in python-dev.

### CODE REVIEW

**Comment by matz-e** [comment]
OK to test

**Review by matz-e — APPROVED** [review]

**Comment by genric** [comment]
@matz-e, @pramodk, @wizmer I'm good to merge this.

File: var/spack/repos/builtin/packages/python-dev/package.py [hunk 1]
```diff
@@ -11,9 +11,9 @@ class PythonDev(Package):
     """Meta package to bundle python packages for development"""
 
     homepage = "http://www.dummy.org/"
-    url      = "https://www.dummy.org/source/dummy-0.1.zip"
+    url      = "https://www.dummy.org/source/dummy-0.2.zip"
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/python-dev/package.py:14]: For future reference, this version change here is not really needed. Spack will be able to guess the version.

**Review by matz-e — APPROVED** [review]
Still looks good to me. Do you have permission to squash & merge yourself?

File: var/spack/repos/builtin/packages/python-dev/package.py (hunk 1 above)
Comment (genric) [reply var/spack/repos/builtin/packages/python-dev/package.py:14]: I guess, but just to keep it consistent :)

**Comment by genric** [comment]
> Still looks good to me. Do you have permission to squash & merge yourself?

Nope, I don't have permissions.

**Comment by matz-e** [comment]
> > Still looks good to me. Do you have permission to squash & merge yourself?
> 
> Nope, I don't have permissions.

Could you try again, please? (@pramodk I added the NSE team with write permissions to avoid this in the future)

**Comment by genric** [comment]
Yep, thanks! i have the powers now :)

**Pull request merged**